### PR TITLE
MK4.2 Part 1.2

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

File: addon/validations/construction-project.js [hunk 1]
```diff
@@ -43,101 +42,7 @@ export default {
...
-      message: "{description} must be on or after {onOrAfter}",
-      allowBlank: true,
-    }),
-    validateDate({
-      onOrBefore: new Date(),
-      message: "{description} must be on or before {onOrBefore}",
-      allowBlank: true,
-    }),
-  ],
+  projectAnnouncementDate: [validatePresence(true)],
```
Comment (luytena) [inline addon/validations/construction-project.js:45]: Did you check that all removed validations are either covered by the quality rules or are no longer needed?

**Review by luytena — COMMENTED** [review]
Otherwise looks good to me :+1:

File: addon/validations/construction-project.js (hunk 1 above)
Comment (velrest) [reply addon/validations/construction-project.js:45]: Noooo :point_right: :point_left: 
Will do that.

Comment (velrest) [reply addon/validations/construction-project.js:45]: Tested, results in open issues on the project

Comment (luytena) [reply addon/validations/construction-project.js:45]: Nice!

**Review by luytena — APPROVED** [review]

**Review by luytena — APPROVED** [review]
:100: :fire: :rocket: :heart:

**Pull request merged**